master/job.go

```diff
@@ -52,14 +52,9 @@ func chunkCandidates(alph lib.Alphabet, l int, n int64) ([][]byte, []int64) {
 // nthCandidateFrom computes the n th candidate password from inp
 func nthCandidateFrom(alph lib.Alphabet, n int64, inp []byte) ([]byte, bool) {
 	l := len(inp)
-	var overflow bool
 	z := lib.BytesToBigInt(alph, inp)
 	z = z.Add(z, big.NewInt(n))
```

Comment:
So on this line, `z.Add(z, big.NewInt(n))` is sufficient. No need for the `z =`, because the result is already stored in `z` by calling `Add` on it.